src/applications/personalization/profile/components/alerts/LoadFail.jsx

```diff
@@ -23,3 +24,54 @@ export default function LoadFail() {
 LoadFail.propTypes = {
   information: PropTypes.string.isRequired,
 };
+
+// Helper function to determine which message to render
+// If no props are passed, return the default failure message
+// If children are passed, render those
+// If a sectionName is passed, render the default failure message with the sectionName
+const renderSingleFieldFailMessage = (sectionName, children) => {
+  if (!children && !sectionName) {
+    return defaultFailureMessage;
+  }
+
+  if (children) {
+    return children;
+  }
+
+  return (
+    <span>
+      We’re sorry. Something went wrong on our end and we can’t load your{' '}
+      {sectionName}. Please try again later.
+    </span>
+  );
+};
+
+// alert for a single field that is shown when that field fails to load
+export const SingleFieldLoadFailAlert = ({ sectionName, children }) => {
+  const message = renderSingleFieldFailMessage(sectionName, children);
+  return (
+    <va-alert status="warning" background-only show-icon>
+      {message}
+    </va-alert>
+  );
+};
+
+// custom props validation function to ensure that either children or sectionName is passed, but not both
+function validateChildrenOrSectionNameProps(props, propName, componentName) {
+  if (props?.sectionName && typeof props?.sectionName !== 'string') {
+    return new Error(
+      `Invalid prop \`${propName}\` supplied to \`${componentName}\`. Expected \`sectionName\` to be of type string, received type of \`${typeof props.sectionName}\`.`,
+    );
+  }
+  if (props.children && props.sectionName) {
+    return new Error(
+      `Invalid prop \`${propName}\` supplied to \`${componentName}\`. Either pass \`children\` or \`sectionName\` but not both. To render default failure message, pass neither prop.`,
```

Comment:
These escaped back-ticks seem weird to me.

Reply:
They are weird lol... that was how I got it to look like the standard prop-types warnings locally, but I'll see if there is a better way around that.